app/controllers/application_controller.rb

```diff
@@ -62,6 +62,7 @@ def user_event_creator
     @user_event_creator ||= UserEventCreator.new(request, current_user)
   end
   delegate :create_user_event, :create_user_event_with_disavowal, to: :user_event_creator
+  delegate :opt_out_rem_me, to: :decorated_session
```

Comment:
Can we clarify this name?

1. I think we should try to use the full name, so  `opt_out_remember_me`?
2. When this is true, this means that we're opting you out of remembering the browser, which means we don't give you the option to remember the browser?

What if we flipped that around: `prevent_remember_me` or `disable_remember_me`? The reason I suggest is that the user isn't given a choice to opt out, it's the SP configs that are actively preventing it.

A quick one-liner to rename it everywhere, if you want:

```bash
git grep -l opt_out_rem_me | xargs perl -p -i -e 's/opt_out_rem_me/disable_remember_me/g'
```

Reply:
Or maybe we negate all the values, and call it `allow_remember_me` or something

Reply:
We are not preventing remember me we are simply not making it the default.

Reply:
And since remember me is the preferred mode we don't want to negate the values otherwise we'll need to set every sp to true (vs a future sp to true if there are any)